bin/build-cluster-catalog.py

```diff
@@ -1,128 +1,215 @@
 #!/usr/bin/env python
 
-"""Build and write out the NGC-star-clusters.fits catalog.
+"""Build and write out the NGC-star-clusters.fits catalog."""
 
-"""
 import os
 import numpy as np
+import pandas as pd
 import numpy.ma as ma
 from astropy.io import ascii
-from astropy.table import Table, vstack
-from astrometry.util.starutil_numpy import hmsstring2ra, dmsstring2dec
-from astrometry.libkd.spherematch import match_radec
-from pkg_resources import resource_filename
-
-#import desimodel.io
-#import desimodel.footprint
-
-#tiles = desimodel.io.load_tiles(onlydesi=True)
-
-if not os.path.isfile('/tmp/NGC.csv'):
-    os.system('wget -P /tmp https://raw.githubusercontent.com/mattiaverga/OpenNGC/master/NGC.csv')
-
-NGC = ascii.read('/tmp/NGC.csv', delimiter=';')
-NGC = NGC[(NGC['RA'] != 'N/A')]
-
-ra, dec = [], []
-for _ra, _dec in zip(ma.getdata(NGC['RA']), ma.getdata(NGC['Dec'])):
-    ra.append(hmsstring2ra(_ra.replace('h', ':').replace('m', ':').replace('s','')))
-    dec.append(dmsstring2dec(_dec.replace('d', ':').replace('m', ':').replace('s','')))
-NGC['ra'] = ra
-NGC['dec'] = dec
-
-objtype = np.char.strip(ma.getdata(NGC['Type']))
+from astropy.table import Table
+from astropy.table import vstack as ap_vstack
+from astropy.coordinates import SkyCoord, match_coordinates_sky
+import astropy.units as u
+from importlib.resources import files
+import sys
+
+sys.path.append("/Users/arielamsellem/Desktop/missing_PNe/legacypipe/")
+
+# import desimodel.io
+# import desimodel.footprint
+# tiles = desimodel.io.load_tiles(onlydesi=True)
+
+# Grab newest NGC catalog and its addendum
+if not os.path.isfile("/tmp/NGC.csv"):
+    os.system(
+        "wget -P /tmp https://raw.githubusercontent.com/mattiaverga/OpenNGC/refs/heads/master/database_files/NGC.csv"
```

Comment:
Please change to https://github.com/mattiaverga/OpenNGC/blob/v20231203/database_files/NGC.csv